Update --help text for broker and lp api. (#370)
chainflip-io/chainflip-docs: pages/lp/integrations/lp-api.mdx

## Command line arguments

- The `state_chain.ws_endpoint` should point at a **synced Chainflip State Chain RPC node**. The default is `ws://localhost:9944` — assuming it is run locally.

> - The `state_chain.signing_key_file` should be the **LP's private key** for their on-chain account. The account should be [funded](/mainnet/validator-setup/funding-and-bidding#adding-funds-to-your-validator-node). The account type should be set to LP. The default is `/etc/chainflip/keys/signing_key_file`.

Reviewer: ```suggestion
- The `state_chain.signing_key_file` should be a path to a file containing the hex-encoded **private key** for the on-chain LP account. The default file path is `/etc/chainflip/keys/signing_key_file`. The account should be [funded](/mainnet/validator-setup/funding-and-bidding#adding-funds-to-your-validator-node) and the account role should be set to LP. 
```

> - The `port` is the port on which the LP will listen for connections. Use `0` to assign a random port. The default is `80`.

Reviewer: ```suggestion
- The `port` is the port on which the LP API will listen for connections. Use `0` to assign a random port. The default is `80`.
```

> - The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP will listen for health check requests. Both arguments have to be specified. If none are, health check functionality is not provided.

Reviewer 2: We'll want to state that these will only be available from version 1.7+ (same below) so it can be merged before we release 1.7 - and also so people can see the upcoming features if they're looking at the docs

> - The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP will listen for health check requests. Both arguments have to be specified. If none are, health check functionality is not provided.

Reviewer 2: ```suggestion
- The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP will listen for health check requests. Both arguments have to be specified for the health check server to start.
```

> - New functionality available from version 1.7+: The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP will listen for health check requests. Both arguments have to be specified for the health check server to start.

Reviewer: ```suggestion
- New functionality available from version 1.7+: The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP API will listen for health check requests. Both arguments have to be specified for the health check server to start.
```

- The `state_chain.ws_endpoint` should point at a **synced Chainflip State Chain RPC node**. The default is `ws://localhost:9944` — assuming it is run locally.
- The `state_chain.signing_key_file` should be the **LP's private key** for their on-chain account. The account should be [funded](/mainnet/validator-setup/funding-and-bidding#adding-funds-to-your-validator-node). The account type should be set to LP. The default is `/etc/chainflip/keys/signing_key_file`.
- The `port` is the port on which the LP will listen for connections. Use `0` to assign a random port. The default is `80`.

> - The `health_check.hostname` and `health_check.port` describe the hostname and port where the LP will listen for health check requests. Both arguments have to be specified for the health check server to start. This functionality is going to be available from version 1.7+.

Reviewer 2: nit: could we put the version sentence at the start, just because people tend not to read things to the end...

    -h, --help
            Print help information

        --health_check.hostname <health_check.hostname>
            Hostname for this LP server's healthcheck. Requires --health_check.port to be present as
            well.

        --health_check.port <health_check.port>
            Port for this LP server's healthcheck. Requires --health_check.hostname to be present as
            well.

        --port <PORT>
            The port number on which the LP server will listen for connections. Use 0 to assign a
            random port. [default: 80]

        --state_chain.signing_key_file <SIGNING_KEY_FILE>

>             A path to a file that contains the LP's secret key for signing extrinsics. [default:

Reviewer 2: 👌
